### PR TITLE
Tidy up the installation instructions

### PR DESCRIPTION
**Pull request by sdake**
We use our Istio.io charts distribution mechanism extensively
in this documentation rework. Helm is a great package manager,
and does a pretty decent job of distributing packages.  These
are not advanced features of helm manifests - but insteead tried
and tested CLI operations that have been around since the dawn
of Helm.

### CODE REVIEW

**Comment by geeknoid** [comment]
Deploy preview for *preliminary-istio* failed.

Built with commit 36cc381be01148ac70705960d203c034b7894dbd

https://app.netlify.com/sites/preliminary-istio/deploys/5c68060130f53500081489fd

**Comment by sdake** [comment]
I am not sure why circleci is failing - perhaps a misplaced tag?  If anyone has any advice, it would be welcome.

Cheers
-steve

**Comment by geeknoid** [comment]
I can't see anything obvious that would trigger the lint warnings. I suggest running 'make lint' locally and fiddling with the contents to see what's going on.

**Comment by sdake** [comment]
@geeknoid I had a `((< /text >}}` instead of `{{< text >}}`.  My vision isn't great, so it's a good thing linters are around.  It took me 30 minutes (using your advice of trying out the linter/rendering) before I spotted the problem :)

Cheers
-steve

**Comment by sdake** [comment]
This change is as simple as I can make the existing documentation.  I suspect we need our own little "helm subitem" that has various other instructions for using Helm beyond what we have in this work.  I also think we need to break the quickstart out into a separate document (and the quickstart is `kubectl apply -f istio-demo.yaml` followed by "get into bookinfo".  I will have some time to do some of this rework next week, however, could use a bit of helmp.

Cheers
-steve

File: content/docs/setup/kubernetes/helm-install/index.md [hunk 1]
```diff
@@ -53,47 +61,65 @@ The following commands have relative references in the Istio directory. You must
...
 
     {{< text bash >}}
-    $ cat @install/kubernetes/namespace.yaml@ > $HOME/istio.yaml
-    $ helm template install/kubernetes/helm/istio --name istio --namespace istio-system >> $HOME/istio.yaml
+
+    $ kubectl create namespace istio-system
+
     {{< /text >}}
 
-    If you want to enable [global mutual TLS](/docs/concepts/security/#mutual-tls-authentication), set `global.mtls.enabled` and `global.controlPlaneSecurityEnabled` to `true` for the last command:
```
Comment (sdake) [inline content/docs/setup/kubernetes/helm-install/index.md]: note I removed this - this doesn't belong in the quickstart, but instead in some more "advanced helm guide" or in the as-of-yet unimplemented profiles guide.

@linsun ^^

**Pull request merged**